docs/content/guides/major-postgres-version-upgrade.md

```diff
@@ -152,6 +152,8 @@ rm -rf '/pgdata/{{< param fromPostgresVersion >}}'
 
 When you are satisfied with the upgrade, you can execute this command to remove the old data directory. Do so at your discretion.
 
+The script does not delete old WAL-files so these must be deleted manually.
```

Comment:
Happy to add this to the documentation, although we would like to word it a little differently. How about the following:
```suggestion
Note that the `delete_old_cluster.sh` script does not delete the old WAL files. These are typically found in `/pgdata/pg{{< param fromPostgresVersion >}}_wal`, although they can be stored elsewhere. If you would like to delete these files, this must be done manually.
```

Reply:
Writing the above comment also makes me realize that line 150 isn't quite right. Could you add an edit to that line to your PR? It should read:

`rm -rf '/pgdata/pg{{< param fromPostgresVersion >}}'`